dartagnan/src/main/java/com/dat3m/dartagnan/wmm/utils/Tuple.java

```diff
@@ -2,25 +2,7 @@
 
 import com.dat3m.dartagnan.program.event.core.Event;
 
-import java.util.function.Function;
-
-public class Tuple implements Comparable<Tuple> {
-
-    private final Event first;
-    private final Event second;
-
-    public Tuple(Event first, Event second) {
-        this.first = first;
-        this.second = second;
-    }
-
-    public Event getFirst(){
-        return first;
-    }
-
-    public Event getSecond(){
-        return second;
-    }
+public record Tuple(Event first, Event second) implements Comparable<Tuple> {
```

Comment:
Still used in SymmetryEncoder and CoSymmetryBreaking. It would be nice to drop it completely.